projects/plugins/jetpack/_inc/client/at-a-glance/scan.jsx

```diff
@@ -271,6 +258,25 @@ class DashScan extends Component {
 		);
 	}
 
+	renderAction( url, message ) {
+		if ( this.props.isAtomicSite ) {
+			return null;
+		}
+
+		return (
+			<Card
+				compact
+				key="manage-scan"
+				className="jp-dash-item__manage-in-wpcom"
+				href={ url }
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{ message }
+			</Card>
+		);
+	}
+
```

Comment:
Moved this function into the class so that the runtime can access the props of the parent component and check whether is it an Atomic site or not.